arch/arm/soc/nordic_nrf/nrf51/dts.fixup

```diff
@@ -1,9 +1,9 @@
 /* SoC level DTS fixup file */
 
 #define CONFIG_NUM_IRQ_PRIO_BITS	ARM_V6M_NVIC_E000E100_ARM_NUM_IRQ_PRIORITY_BITS
-#define CONFIG_UART_NRF5_IRQ_PRI	NORDIC_NRF_UART_40002000_IRQ_0_PRIORITY
-#define CONFIG_UART_NRF5_BAUD_RATE	NORDIC_NRF_UART_40002000_CURRENT_SPEED
-#define CONFIG_UART_NRF5_NAME		NORDIC_NRF_UART_40002000_LABEL
+#define CONFIG_UART_NRFX_IRQ_PRI	NORDIC_NRF_UART_40002000_IRQ_0_PRIORITY
+#define CONFIG_UART_NRFX_BAUD_RATE	NORDIC_NRF_UART_40002000_CURRENT_SPEED
+#define CONFIG_UART_NRFX_NAME		NORDIC_NRF_UART_40002000_LABEL
```

Comment:
`#define CONFIG_UART_0_IRQ_PRI` and so on.

Reply:
fixed